__tests__/app.test.js

```diff
@@ -42,12 +42,49 @@ describe("/api/topics", () => {
 });
 
 describe("/api", () => {
-    test("GET 200: Should return the updated endpoints.json file with information about avialble endpoints", () => {
+    test("GET 200: Should return the updated endpoints.json file with information about avialable endpoints", () => {
         return request(app)
         .get("/api")
         .expect(200)
         .then((response) => {
-            expect(JSON.parse(response.text)).toEqual(endpoints)
+            expect(JSON.parse(response.text)).toEqual({ endpoints} )
         })
     })
+})
+
+describe("/api/articles/:article_id", () => {
```

Comment:
We should expect back the exact values associated with article number 3 here - not any data!